client/pages/sections/protocols/legacy-animals.js

```diff
@@ -4,10 +4,10 @@ import Fieldset from '../../../components/fieldset';
 import Repeater from '../../../components/repeater';
 import ReviewFields from '../../../components/review-fields';
 
-const LegacyAnimal = ({ updateItem, values, readonly, prefix, fields, index }) => (
+const LegacyAnimal = ({ updateItem, values, readonly, prefix, fields, index, editable }) => (
   <Fragment>
     {
-      !readonly
+      (!readonly && editable)
```

Comment:
Why are these two not equivalent?

Either it's editable _or_ it's read-only surely?

Reply:
readonly is at an application level, editable is at a state level - ie it can be not in edit mode (readonly = false) but check your answers (editable = false)

Reply:
👍